Touch Bar Simulator/ToolbarSlider.swift

```diff
@@ -0,0 +1,57 @@
+//
+//  ToolbarSlider.swift
+//  Touch Bar Simulator
+//
+//  Created by Wayne Yeh on 2017/3/20.
+//  Copyright © 2017年 Sindre Sorhus. All rights reserved.
+//
+
+import Cocoa
+
+private class ToolbarSliderCell: NSSliderCell {
+	
+	class var knob: NSImage {
+		let frame = NSRect(x: 0, y: 0, width: 32, height: 32)
```

Comment:
Add a `print("draw")` inside here and you'll see that it's not just drawn once, but many times when dragging.